projects/packages/external-media/src/shared/sources/jetpack-app-media.js

```diff
@@ -72,6 +73,19 @@ function JetpackAppMedia( props ) {
 		: 'jetpack-external-media-wrapper__jetpack_app_media-wrapper has-no-image-uploaded';
 
 	const selectButtonText = selectedImages => {
+		if ( isCopying ) {
+			return sprintf(
+				/* translators: %1$d is the number of images that were selected. */
+				_n(
+					'Inserting… %1$d image',
```

Comment:
Do we need to change this to `Inserting xxx images....`?

Also, how to test this? I can't see this "provider" 🤔

Reply:
You have to go to the editor > insert image block > select image > Your phone, and then upload the photos from your app.

Reply:
Ohh nice feature 😂 though I still don't see the copy ("Inserting...") when I select the image from the app

Reply:
I found the feature is only enabled when we allow people to select multiple images from the external media modal. So, you have to use blocks like `Story` to see it 😓